src/lazy/lazy.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <folly/Lazy.h>
+// #include <folly/ConcurrentLazy.h>
+
+using namespace folly;
+
+int main() {
+  auto val = lazy([&]{
+    return UINT16_MAX;
+  });
+  val();
+
+  // auto const cval = concurrent_lazy([&]{
+  //   return UINT32_MAX;
+  // });
+  // cval();
```

Comment:
concurrent lazy should be in a different file and PR

Reply:
Yes I was just trying it out, I think pretty much the same formatter will work. I couldn't get this to compile though, it gives a difficult to understand error...

Reply:
Let me know if you need help compiling it